shell.nix

```diff
@@ -1,8 +1,19 @@
-with import <nixpkgs> { };
+let
+  moz_overlay = import (builtins.fetchTarball
+    "https://github.com/mozilla/nixpkgs-mozilla/archive/9b11a87c0cc54e308fa83aac5b4ee1816d5418a2.tar.gz");
+  nixpkgs = import <nixpkgs> { overlays = [ moz_overlay ]; }; # 23.11
```

Comment:
This comment isn't very clear, and I'm not sure it would add much even if clarified.